core/base/src/constants/platforms.ts

```diff
@@ -0,0 +1,119 @@
+import { ChainName } from "./chains";
+import {
+  column,
+  toMapping,
+  toMappingFunc,
+  reverseArrayMapping,
+} from "../utils/mapping";
+
+const platformAndChainsEntries = [
```

Comment:
Oof, need to make terminology consistent across repo

Reply:
sorry which term? platform?

Reply:
Yes, Platform.  I think Andreas has been using that term which is ok with me.  But I had been using the term `Context`, and I know there's been other instances of us using different terminology for the same things

Reply:
'Context' is too much of a generic term in programming to use for platforms (which is also questionable).
Imho it's similar to 'environment' where it's natural to jump to 'environment variable' etc.

Reply:
Maybe ChainFamily would be better because it is less generic and also reasonable descriptive...

but since Ben and I agreed that we'd rename Chain, Platform, Module, ... to ChainName, PlatformName, ModuleName, this would then turn it into ChainFamilyName... :grimacing:

Reply:
(and `Module` is a terrible name for the same reason - we should really come up with something better if we can)